doctors_surgery.html

```diff
@@ -26,17 +26,17 @@
     </nav>
 
     <label for="site-search">Search...</label>
-    <input type="search" name="search" aria-label="Search through site content">
+    <input type="search" id="site-search" name="search" aria-label="Search through site content">
     <button>Search</button>
 
     <section>
-        <h3><strong><em>COVID RESTRICTION UPDATE</em></strong></h3>
+        <h3><strong>COVID Restriction Update</strong></h3>
 
         <p>Throughout the pandemic we have strove to make the surgery as safe as possible for all our patients and
             staff, we will continue to do so.</p>
 
-        <p>From the <strong>19TH JULY 2021 NOTHING WILL CHANGE AT THE SURGERY</strong> – Patients will still need to
-            <em>wear a mask, socially distance and use hand sanitiser</em> in the surgery and staff will continue to
+        <p>From the <strong>19th July 2021 nothing will change at the Surgery</strong> – Patients will <em>still need</em> to
```

Comment:
In terms of this bit of text, should the emphasis be on the fact that some expectations will still be in place? My thinking was if I was reading it out loud, which bit of text would I emphasise with my voice?